### PR TITLE
trailing stop-loss rule

### PR DESCRIPTION
**Pull request by Chr1st0ph**
Changes proposed in this pull request:
- add a trailing stop-loss rule ( similar to StopLossRule )
- added an entry to the unreleased section of `CHANGES.md`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Changelog for `ta4j`, roughly following [keepachangelog.com](http://keepachangelog.com/en/1.0.0/) from version 0.9 onwards.
 
+## 0.12-SNAPSHOT (unreleased)
+
+## Added
+- **TrailingStopLossRule**: new rule that is satisfied if trailing stop loss is reached
```
Comment (team172011) [inline CHANGELOG.md]: Where are all the other entries of the 0-12 section from https://github.com/ta4j/ta4j/blob/develop/CHANGELOG.md ?

**Comment by Chr1st0ph** [comment]
I messed it up. I will close this pull request, and make a new one

**Comment by team172011** [comment]
I see the reason was that you wanted to pull into `master` branch, please make PR to `develop `branch

**Comment by Chr1st0ph** [comment]
yes, one of the reasons :)

**Pull request closed without merging**